cli/assets/terraform/test_files

```diff
@@ -0,0 +1,88 @@
+./aws/codebuild/codebuild_project/project_encryption/tests/test.yml
```

Comment:
I'd love to see something in the documentation about this file. Also, ABC sorting would be great too to make it easier for adding or editing tests in the future.

Reply:
removed this. this file wasnt intentionally added :grimacing: